Detectors/MUON/MCH/Raw/Common/src/RDHManip.cxx

```diff
@@ -138,7 +198,8 @@ RAWDataHeaderV4 createRDH(uint16_t cruId, uint8_t linkId, uint16_t solarId, uint
   uint16_t memorySize = payloadSize + sizeof(rdh);
 
   rdh.cruID = cruId;
-  rdh.feeId = solarId;
+  rdhLinkId(rdh, linkId);
```

Comment:
If you change that, then the current implementation of `rdhLinkId()` would not work any longer, right?
because it expects a `linkId` between 0 and 23, see https://github.com/aphecetche/AliceO2/blob/mch-raw-data-decoders-reorganization/Detectors/MUON/MCH/Raw/Common/src/RDHFieldsV4.cxx#L42

Reply:
It might indeed require changes in a few locations, but I believe we should stick to the convention of using (endpoint 0..1, linkid 0..11) everywhere (as is/will be now the case for all FLP scripts, right ?) to avoid confusion. Do you agree ?

Reply:
I agree. Does it mean you will add an `endpointId` parameter to `rdhLinkId()` function?
By the way, I guess this function is only used in unit tests or for MC data encoding, in which case one needs to generate "synthetic" RDHs?